reactivesocket-tck-drivers/src/main/java/io/reactivesocket/tckdrivers/test/PayloadImpl.java

```diff
@@ -0,0 +1,49 @@
+package io.reactivesocket.tckdrivers.test;
+
+import io.reactivesocket.Payload;
+
+import java.nio.ByteBuffer;
+import java.nio.charset.StandardCharsets;
+
+public class PayloadImpl implements Payload // some JDK shoutout
```

Comment:
maybe for a separate change, but I believe this move allows us to use io.reactivesocket.util.PayloadImpl